src/showcase/index.md

```diff
@@ -6,20 +6,24 @@ show-nav-get-started-button: true
 toc: false
 ---
 
-<section class="landing-page__hero">
-    <div class="container">
-        <div class="row">
-            <div class="col-md-6 col-lg-4 offset-md-5 offset-lg-7">
-                <div class="card">
-                    <div class="card-body">
-                        <h2>Featured</h2>
-                        <h3>Hamilton Musical</h3>
-                        <p>
-                        Official app of the hit Broadway musical, Hamilton. Includes daily lotteries,  exclusive news and
-                        videos, a trivia game, merchandise store, and more.
-                        </p>
-                        <a class="btn btn-link" href="#">Learn more</a>
-                        <button class="btn btn-link btn-icon" type="button" data-toggle="modal" data-target="#videoModal"><i class="material-icons">play_circle_filled</i> Watch the video</button>
+<section class="landing-page__hero showcase__carousel">
+    <div class="showcase__carousel__slide" style="background-image: url('{% asset 'showcase/hero-hamilton.jpg' @path %}') ">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-7 offset-md-5 col-lg-5 offset-lg-6 col-xl-4 offset-xl-7">
+                    <div class="card">
+                        <div class="card-body">
+                            <h2 class="card-category">Featured</h2>
```

Comment:
As a hero, it seems obvious to me that this is a featured app. Having the "Featured" label breaks the hero card aesthetics IMHO. Could we drop it?